src/VernaForm.tsx

```diff
@@ -1,24 +1,32 @@
 import Form from '@rjsf/core';
 import { useVerna } from './context/VernaContextProvider';
-import RenderEditorFieldTemplate from './RenderMethods/RenderEditorFieldTemplate';
+import RenderFieldTemplate from './RenderMethods/RenderFieldTemplate';
 
 interface VernaFormProperties {
   onSubmit: (formData: unknown) => void;
 }
 
 function VernaForm({ onSubmit }: VernaFormProperties) {
-  const { schema, uiSchema, widgets, selectedFormData, handleSubmit } = useVerna();
+  const { schema, uiSchema, widgets, selectedFormData, handleSubmit, isEditor } = useVerna();
 
   return (
     <Form
+      name="verna-form"
+      tagName={isEditor ? 'div' : undefined}
       className="form"
       schema={schema}
       uiSchema={uiSchema}
       formData={selectedFormData}
       onSubmit={handleSubmit(onSubmit)}
       widgets={widgets}
-      FieldTemplate={RenderEditorFieldTemplate}
-    />
+      disabled={isEditor}
+      FieldTemplate={RenderFieldTemplate}
+      liveValidate={!isEditor}
+    >
+      <button type="submit" name="verna-form">
+        Save
+      </button>
```

Comment:
Those 3 lines are not that usefull because if i remove it,it's replaces with a default submit button, i only use it for customization purpose, we would need to use a style to hide it, or wrap it in a div or whatever and not display the button if in editor mode.
I searched a bit for it, and there is a uiSchema instruction to customize it, hide and change text, but it seems a bit working on, it works on the playground but can't find how to make it work in verna, but i find a recent commit (13 april), that seems to add it
https://github.com/rjsf-team/react-jsonschema-form/commit/57f00cb522c698a2a201c7204d78dfc6323ac6f6
So i would suggest to wait a bit for the next release to add this fix, we can just disable it for now and do it a clean way once it's release?

nb: i checked the sources of the lib in node_modules, the files managing this truly doesn't exists, with the next release it will be really nice to customize and every client will be able to customize it easily